src/coreclr/src/ilasm/asmman.cpp

```diff
@@ -965,6 +965,7 @@ HRESULT AsmMan::EmitManifest()
                 {
                     wcscpy_s(wzFileName,2048,pwzInputFiles[j]);
                     pwz = wcsrchr(wzFileName,'\\');
+                    if(pwz == NULL) pwz = wcsrchr(wzFileName,'/');
```

Comment:
We should check on Windows for `\\` only and on Unix for `/` only. Can you please use PATH_SEPARATOR_CHAR_A define here? That is defined to `\\` on Windows and `/` on Unix.

Reply:
I've fixed all references to use DIRECTORY_SEPARATOR_CHAR_A as needed.